tests/index.js

```diff
@@ -195,6 +214,100 @@ module.exports = {
       .to
       .match(/\w/);
   },
+  'Clicking on logo should open corresponding link': function (browser){
+    const stepOptions = getStepOptions('logoLink', browser);
+    const hexoData = hexo.locals.get('data');
+    const { navigation } = hexoData;
+
+
+    browser
+      .url(path.join(browser.launchUrl, stepOptions.path))
+      .click(stepOptions.logoSelector)
+      .waitForElementVisible(rootTag, 1000)
+      .assert
+      .urlEquals(browser.launchUrl + navigation.logo.path, 'Clicking on logo should open corresponding link.');
+  },
+  'page should scroll to heading, when clicking on toc': function (browser){
+    const stepOptions = getStepOptions('scrollToAnchor', browser);
+
+    const eHeader = `${stepOptions.element}#${stepOptions.id}`;
+
+    browser
+      .url(path.join(browser.launchUrl, stepOptions.path))
+      .waitForElementVisible(rootTag, 1000)
+      .click('.doc-sidebar a[href="#Quick-Start"')
+      .pause(1000)
+      .getLocationInView(eHeader, (result) => {
+        browser.assert.equal(typeof result, 'object');
+        browser.assert.equal(result.status, 0);
+        browser.assert.equal(result.value.y, 0, 'anchor header should be at the top of the page view');
+      });
+
+  },
+  'when resizing window to mobile view, sidebar should collapse': function (browser){
+    const stepOptions = getStepOptions('scrollToAnchor', browser);
+
+    browser
+      .url(path.join(browser.launchUrl, stepOptions.path))
+      .waitForElementVisible(rootTag, 1000)
+      .resizeWindow(mobileSize.x, mobileSize.y)    // iPhone 6 specs
+      .assert
+      .cssProperty('.doc-sidebar', 'transform', 'matrix(1, 0, 0, 1, -230, 0)');  // TODO: Check if better way to this
```

Comment:
There is a better way to check this? If so do it and remove the comment, if not update the comment explaining what are you doing